### PR TITLE
III-5108 Add guide about contributors

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Guide about contributors

---

Ticket: https://jira.uitdatabank.be/browse/III-5108

Note: This is still under development (but for the biggest part already ready for deploy), but we can already merge this page in the meantime and just wait to publish the docs  until the new endpoints are deployed. And these docs are not actively used yet in a large scale anyway.

### CODE REVIEW

**Comment by corneelwille** [comment]
I will review this when I'm testing the API functionality

File: projects/uitdatabank/docs/entry-api/shared/contributors.md [hunk 1]
```diff
@@ -1,7 +1,124 @@
...
+Content-Type: application/json
+
+[
+  "john.doe@acme.org",
+  "jane@acme.com",
+  "smith@ipsum.be"
+]
+```
+
+## Removing contributors
```
Comment (corneelwille) [inline projects/uitdatabank/docs/entry-api/shared/contributors.md:56]: This did not work while testing  (see test results III-5214), but I assume that is a bug in the code or a mistake from my end.

**Review by corneelwille — APPROVED** [review]
Nice & clear documentation.

File: projects/uitdatabank/docs/entry-api/shared/contributors.md (hunk 1 above)
Comment (bertramakers) [reply projects/uitdatabank/docs/entry-api/shared/contributors.md:56]: That is indeed a bug I think

**Pull request merged**